packages/suite/src/storage/definitions.ts

```diff
@@ -60,7 +60,7 @@ export interface SuiteDBSchema extends DBSchema {
     };
     devices: {
         key: string;
-        value: AcquiredDevice;
+        value: Omit<AcquiredDevice, 'path'>;
```

Comment:
it looks like that device saving logic always enforced device with `path = ''`.
I am not sure whether I should stick to this or I may safely assume that we don't want to save path at all. Maybe @szymonlesisz knows?

Reply:
ok, it plays some role, at least here https://github.com/trezor/trezor-suite/blob/develop/suite-common/wallet-core/src/device/deviceReducer.ts#L133 so I am going to put it back